maps/away/away_site/automated_waystation/automated_waystation.dm

```diff
@@ -0,0 +1,98 @@
+/datum/map_template/ruin/away_site/automated_waystation
+	name = "Automated Waystation"
+	description = "Automated Waystation."
+	unit_test_groups = list(1)
+
+	id = "automated_waystation"
+	spawn_cost = 1
+	spawn_weight = 1
+	prefix = "away_site/automated_waystation/"
+	suffixes = list("automated_waystation.dmm")
+
+	sectors = list(ALL_COALITION_SECTORS)
```

Comment:
this is effectively a different variant of the current sensor relay, and it is redundant if both spawn at the same time

so either:
- make this station guaranteed, and use the `ban_ruins` var on both to ban the other one (imo this is better, but make sure it actually works in game)
- make this station guaranteed, and blacklist the current sensor relay from spawning in the coc